### ISSUE TITLE
Screenshot button does not appear in Gradio anymore

### ISSUE THREAD

**Issue by osanseviero** [gradio-app/gradio#733]
### Describe the bug

I cannot see Gradio screenshot button. 

### Reproduction

Examples
* https://huggingface.co/spaces/osanseviero/gradio_test
* https://huggingface.co/spaces/Salesforce/BLIP

Example code

```python
!pip install transformers gradio sentencepiece
import gradio as gr

from transformers import pipeline

pipe = pipeline("translation", model="Helsinki-NLP/opus-mt-en-es")

def predict(text):
  return pipe(text)[0]["translation_text"]
  
title = "Interactive demo: Helsinki-NLP English to Spanish Translation"

iface = gr.Interface(
  fn=predict, 
  inputs=[gr.inputs.Textbox(label="text", lines=3)],
  outputs='text',
  title=title,
  examples=[["Hello! My name is Omar"], ["I like this workshop"]]
)

iface.launch(debug=True, share=True)
```

### System Info

```shell
Gradio version 2.8.4
Got issues both in Colab and Spaces
```

### Severity

critical

**Comment by FarukOzderim** [comment]
Thank you for the report, I presume the screenshot button was available in the previous versions for submitting an image?

**Comment by osanseviero** [comment]
Yes. Here is an example of famous space which I'm sure we had screenshot button before. https://huggingface.co/spaces/akhaliq/ArcaneGAN

**Comment by abidlabs** [comment]
We no longer support the screenshot button because it was super buggy and hard to maintain. But we should make this clear to the users. Let's:

* Set `allow_screenshot` to be `False` by default
* And print a deprecation warning to users if they try to set it to `True`

**Comment by abidlabs** [comment]
Closed by #739, thanks @FarukOzderim!
